_posts/2019-05-28-improved-hive-bucketing.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title:  "Improved Hive Bucketing"
+author: David Phillips
+---
+
+The next release of Presto (version 312) adds support for the more
```

Comment:
I you wait until after it's released, you can link to the release notes here and just say "Presto 312 adds ..."